### PR TITLE
fix: Fix Helm Acceptance Tests

### PR DESCRIPTION
**Pull request by xpuska513**
## What
* Migrate from KIND to K3s setup to improve test execution time
* Fix all possible issues that cause for ac tests to fail
* Update umbrella chart values.yaml to reference latest changes made in kustomize so kube config == helm config both dev and stable

### CODE REVIEW

File: .github/workflows/gradle.yml
Comment (git-phu) [inline .github/workflows/gradle.yml:739]: @xpuska513 did we need this for tests to work?

Comment (xpuska513) [reply .github/workflows/gradle.yml:739]: nah, I can revert it back to the default value, but I saw that the same value was set in kube AC tests so did the same here.

File: .github/workflows/gradle.yml
Comment (supertopher) [inline .github/workflows/gradle.yml:963]: and this is fine and dandy, but I think there is a tmate action that does more or less all of this

File: .github/workflows/gradle.yml
Comment (supertopher) [inline .github/workflows/gradle.yml:992]: Can we do more to inform the user about this failure?
```suggestion
        run: kubectl get pods -A || echo "failed to get pods!"
```

File: .github/workflows/gradle.yml
Comment (supertopher) [inline .github/workflows/gradle.yml:1039]:
```suggestion
```

File: tools/bin/acceptance_test_kube_helm.sh
Comment (supertopher) [inline tools/bin/acceptance_test_kube_helm.sh:74]: I genuinely don't know.  is Debug too noisy for our users or is there useful feedback there?

**Review by supertopher — COMMENTED** [review]
this looks solid to me.  I'd like an approval from someone with more helm context though

File: .github/workflows/gradle.yml
Comment (git-phu) [inline .github/workflows/gradle.yml:13]: let's default to "false"

File: .github/workflows/gradle.yml
Comment (git-phu) [reply .github/workflows/gradle.yml:741]: @xpuska513  I think this is already set in https://github.com/airbytehq/airbyte/blob/34d4a71875e5413cee063fd301dc9b406686238b/gradle.properties#L4 so we shouldn't need to specify this here

File: .github/workflows/gradle.yml
Comment (git-phu) [inline .github/workflows/gradle.yml:892]: do we need a different ami from the default one set in the action?
https://github.com/airbytehq/airbyte/blob/34d4a71875e5413cee063fd301dc9b406686238b/.github/actions/start-aws-runner/action.yml#L12
If so, what's different about this ami?

File: .github/workflows/gradle.yml
Comment (git-phu) [inline .github/workflows/gradle.yml:1003]: same thing, was this for testing too or did this help somehow?

File: .github/workflows/gradle.yml
Comment (git-phu) [inline .github/workflows/gradle.yml:1005]: shouldn't need this since it's in gradle.properties already

File: tools/bin/acceptance_test_kube_helm.sh
Comment (git-phu) [reply tools/bin/acceptance_test_kube_helm.sh:74]: yeah i think we should remove `LOG_LEVEL=DEBUG`
Can always add it back for debugging a specific job
but it would add a lot of noise by default

**Review by git-phu — COMMENTED** [review]
great work!

looks like it's working now, and faster too!

Just had a few comments.  Should be good to merge after that!

File: .github/workflows/gradle.yml
Comment (xpuska513) [reply .github/workflows/gradle.yml:13]: Hmm, i thought it was on false :) need to revisit the if statement then, since its not working properly.

File: tools/bin/acceptance_test_kube_helm.sh
Comment (xpuska513) [reply tools/bin/acceptance_test_kube_helm.sh]: Done

File: .github/workflows/gradle.yml
Comment (xpuska513) [reply .github/workflows/gradle.yml]: Done

File: .github/workflows/gradle.yml
Comment (xpuska513) [reply .github/workflows/gradle.yml]: Done

File: .github/workflows/gradle.yml
Comment (xpuska513) [reply .github/workflows/gradle.yml:889]: https://github.com/airbytehq/airbyte/blob/master/.github/workflows/gradle.yml#L685

File: .github/workflows/gradle.yml
Comment (xpuska513) [reply .github/workflows/gradle.yml]: Done

File: .github/workflows/gradle.yml
Comment (xpuska513) [reply .github/workflows/gradle.yml:959]: Tmate actions take runtime space during workflow execution, while this method of running tmate just spins the daemon and we can access the instance during test execution. As for flaws, there's only one, when workflow dies, connection does too.

File: tools/bin/acceptance_test_kube_helm.sh
Comment (Mykyta-Serbynevskyi) [inline tools/bin/acceptance_test_kube_helm.sh:89]: Should it be inside `if` statement above?

File: charts/airbyte/values.yaml.test
Comment (Mykyta-Serbynevskyi) [inline charts/airbyte/values.yaml.test:1395]: nit: newline

File: charts/airbyte/Chart.yaml.test
Comment (Mykyta-Serbynevskyi) [inline charts/airbyte/Chart.yaml.test:67]: nit: newline

File: tools/bin/acceptance_test_kube_helm.sh
Comment (xpuska513) [reply tools/bin/acceptance_test_kube_helm.sh:89]: It's the same as with kube ac tests, idk, but this command doesn't do anything harmful so we can leave it there

**Review by git-phu — APPROVED** [review]
excellent work!

I think this looks fine to merge now

**Comment by xpuska513** [comment]
i'll wait for one more merge before going all in :)

**Review by c-p-b — APPROVED** [review]

File: .github/workflows/gradle.yml
Comment (davinchia) [reply .github/workflows/gradle.yml:12]: this is pretty nice

**Pull request merged**
